src/applications/hca/config/form.js

```diff
@@ -674,8 +691,11 @@ const formConfig = {
               formData.maritalStatus.toLowerCase() === 'separated'),
           uiSchema: {
             'ui:title': 'Spouse’s information',
-            'ui:description':
-              'Please fill this out to the best of your knowledge. The more accurate your responses, the faster we can process your application.',
+            'ui:description': PrefillMessage,
```

Comment:
I don't think there's any prefilled info on this page. We don't have anywhere to get spouse info from.

Reply:
Prefill-message removed.